source/auditing-logging.txt

```diff
@@ -12,17 +12,77 @@ Auditing and Logging
    :depth: 1
    :class: onecol
 
-Intro statement
+To monitor and log |service| platform activities, use auditing and logs.
 
 {+service+} Features and Best Practices for Auditing and Logging
 ----------------------------------------------------------------
 
-Content here
+Auditing
+~~~~~~~~
+
+Database auditing lets administrators track system activity for 
+deployments with multiple users. |service| administrators can select 
+the actions, database users, |service| roles, and LDAP groups that they 
+want to audit. You can configure manual :manual:`auditing </core/auditing>` 
+in |service| of most of the documented :manual:`system event actions 
+<audit-action-details-results>`.
+
+You can :atlas:`set up database auditing <set-up-database-auditing>`.
+
+To learn about best practices for auditing the actions of temporary
+database users, see :atlas:`Audit Temporary Database Users <audit-temp-db-users>`.
+
+By default, |service| performs database authentication auditing in M10+ clusters
+to record authentication events, which include:
+
+- database users
+- source IP addresses
+- timestamps for successful and failed attempts.
+
+In addition, you can configure a JSON-formatted audit filter
+to customize MongoDB auditing.
+For a full list of events you can configure for auditing, and for a list
+of examples, see :manual:`MongoDB auditing </core/auditing>`.
```

Comment:
RE: the above, consulting this list might be a good way to get an idea of how we can rewrite the list to describe events.